### DISCUSSION TITLE
EDGE中使用远程RPC无法连接？

### DISCUSSION THREAD

**Discussion by Jeerui** [alexhua/Aria2-for-chrome#140] [Q&A] [answered]
目前EDGE重使用插件连接远程RPC，提示401 unauthorized
远程服务器使用用户名密码登陆，未使用secret选项，打开插件无登陆框，为何？
手机版的Aria2NG可以正常连接服务器

A 401 unauthorized message is displayed when Explorer is used to connect a remote RPC with EDGE
The remote server logs in with the user name and password, the secret option is not used, and there is no login box when the plug-in is opened. Why?
The mobile version of Aria2NG can connect to the server

**Comment by alexhua** [comment, marked as answer]
用户名密码的方式已经淘汰了，扩展这边不支持。推荐改为secret key验证方式。
